ButterflyMX Demo/Controllers/LoginViewController.swift

```diff
@@ -76,6 +68,23 @@ class LoginViewController: UITableViewController {
                    clientID: Bundle.main.object(forInfoDictionaryKey: "butterflymx-CLIENTID") as? String ?? "N/a")
         SVProgressHUD.setDefaultStyle(.light)
     }
+    
+    private func getBMXAuthProvider(for environment: BMXBackendEnvironment) -> BMXAuthProvider {
+        switch environment {
+        case .development:
+            UserDefaults.standard.set("development", forKey: "environment")
```

Comment:
if you name the function  with 'get', then I think you should not set here anything